flexmeasures/data/tests/test_queries.py

```diff
@@ -85,13 +88,16 @@ def test_collect_power(db, app, query_start, query_end, num_values, setup_test_d
         ),
     ],
 )
-def test_collect_power_resampled(
+def tesfijfijft_collect_power_resampled(
```

Comment:
This looks like some extra characters made their way in this function name, rendering it a non-test function :(

Reply:
Thanks. This test actually uncovered a bug in how I deprecated the `most_recent_only` argument. For future reference, whenever we deprecate an optional argument, we have to change its default value to None. Otherwise, our `replace_deprecated_argument` util function will perceive the deprecated argument as being set to some non-None value.